### ISSUE TITLE
Tachyphagia

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10310]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PsychologyDictionary.org : A pathological form of rapid eating.

PMID: 34487332 " Obesity is a public health problem worldwide. The eating habits of French workers need to be clarified. In particular, **tachyphagia** (meal lasting < 15 min) promotes weight gain. The present study aimed to investigate the presence of tachyphagia at lunchtime, the factors associated with **tachyphagia**, and the relationship between tachyphagia and obesity among workers. 

PMID: 28011862 Esophageal food impaction is more likely to occur on American holidays and national athletic events and is associated with large meals. Patients with intrinsic risk factors should be advised to modify their diet during cultural events associated with tachyphagia and large meals to prevent esophageal food impaction.

PMID: 34748075 Eating behavior disorders were observed in 45% (18/40) of all patients and in 68% (16/23) of overweight/ obese patients. Hyperphagia was frequently found and sometimes associated with **tachyphagia** and persistent absence of satiety

**Parent term (use hpo.jax.org/app)**
Abnormal eating behavior HP:0100738

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:647799  MYT1L-related developmental delay-intellectual disability-obesity syndrome

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000352
- label: Tachyphagia
- parents: [Abnormal eating behavior (HP:0100738)](https://hpo.jax.org/app/browse/term/HP:0100738)
- pmid: PMID:34487332|PMID:28011862
- def: Excessively rapid eating of food.
- comment: Although there is no universally accepted definition of what constitutes excessively rapid eating, one publication states that a meal lasting less than 15 minutes represents tachyphagia. The term also comprises bolting of food, which refers to eat food so quickly that one hardly chews or tastes it.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10310
- synonyms:
